### PR TITLE
US_3_Tsk_112: Add search box for message list view

### PR DESCRIPTION
**Pull request by Alsha-001**

### CODE REVIEW

File: ui/src/Components/feedback/Feedback.js [hunk 1]
```diff
@@ -79,14 +80,15 @@ const Feedback = ({feedbackType, managerList}) => {
...
-      variables: {employeeId: loggedInUser.employeeId},
+      variables: { employeeId: loggedInUser.employeeId },
     });
   };
 
   return (
     <div className="fb-main">
       <nav className="fb-navigation-bar">
         <div className={classes.root}>
+          <SearchBox></SearchBox>
```
Comment (mohantysabyasachi27) [inline ui/src/Components/feedback/Feedback.js]: Add this to Thread.js  grid layout.  It makes more sense to have it inside Thread component.
<SearchBox></SearchBox> can be written as <SearchBox />

File: ui/src/Components/threadview/SearchBox.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    const classes = useStyles();
+    return (
+        <div>
+            <List className="send-container" dense={false}>
+                <ListItem className={classes.root}>
+                    <ListItemText>
+                        <TextField
+                            size="medium"
+                            fullWidth
+                            className="send-input"
```
Comment (mohantysabyasachi27) [inline ui/src/Components/threadview/SearchBox.js]: className is not used.

File: ui/src/Components/threadview/SearchBox.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        root: {
+            width: '100%',
+            maxWidth: '100%',
+            backgroundColor: theme.palette.background.paper,
+        },
+    }));
+    const classes = useStyles();
+    return (
+        <div>
+            <List className="send-container" dense={false}>
```
Comment (mohantysabyasachi27) [inline ui/src/Components/threadview/SearchBox.js]: List is not needed. A simple FormControl and a TextField is all you need.

File: ui/src/Components/feedback/Feedback.js (hunk 1 above)
Comment (Alsha-001) [reply ui/src/Components/feedback/Feedback.js:91]: Search Box will be present in the application even if there are no feedback messages. So will it not be better if we keep it under Feedback.js?

File: ui/src/Components/threadview/SearchBox.js (hunk 2 above)
Comment (Alsha-001) [reply ui/src/Components/threadview/SearchBox.js]: Fixed

File: ui/src/Components/threadview/SearchBox.js (hunk 3 above)
Comment (Alsha-001) [reply ui/src/Components/threadview/SearchBox.js]: Changed to TextField

**Pull request merged**